eq-publisher/src/eq_schema/Answer.js

```diff
@@ -17,8 +17,11 @@ class Answer {
         this.buildNumberValidation(maxValue, "max_value");
       } else if (answer.type === "Date") {
         const { earliestDate, latestDate } = answer.validation;
-        this.buildDateValidation(earliestDate, "minimum");
-        this.buildDateValidation(latestDate, "maximum");
+        Object.assign(
+          this,
+          Answer.buildDateValidation(earliestDate, "minimum")
```

Comment:
I think this can become:
`this.minimum = Answer.buildDateValidation(earliestDate)` and something similar for latestDate